src/applications/personalization/profile/actions/personalInformation.js

```diff
@@ -0,0 +1,148 @@
+import appendQuery from 'append-query';
+import set from 'lodash/set';
+import capitalize from 'lodash/capitalize';
+
+import { PERSONAL_INFO_FIELD_NAMES } from '@@vap-svc/constants';
+
+import {
+  VAP_SERVICE_TRANSACTION_REQUESTED,
+  VAP_SERVICE_TRANSACTION_REQUEST_SUCCEEDED,
+  VAP_SERVICE_TRANSACTION_REQUEST_FAILED,
+  clearTransaction,
+} from '@@vap-svc/actions';
+
+import { getData } from '../util';
+
+import recordEvent from '~/platform/monitoring/record-event';
+import { apiRequest } from '~/platform/utilities/api';
+
+export const FETCH_PERSONAL_INFORMATION = 'FETCH_PERSONAL_INFORMATION';
+export const FETCH_PERSONAL_INFORMATION_SUCCESS =
+  'FETCH_PERSONAL_INFORMATION_SUCCESS';
+export const FETCH_PERSONAL_INFORMATION_FAILED =
+  'FETCH_PERSONAL_INFORMATION_FAILED';
+
+export function fetchPersonalInformation(forceCacheClear = false) {
+  return async dispatch => {
+    dispatch({ type: FETCH_PERSONAL_INFORMATION });
+
+    const baseUrl = '/profile/personal_information';
+
+    const url = forceCacheClear
+      ? appendQuery(baseUrl, { now: new Date().getTime() })
+      : baseUrl;
+
+    const response = await getData(url);
+
+    if (response.errors || response.error) {
+      dispatch({
+        type: FETCH_PERSONAL_INFORMATION_FAILED,
+        personalInformation: { errors: response },
+      });
+      return;
+    }
+
+    // preferred name returns as ALL CAPS, so it needs to be capitalized appropriately for display
+    // TODO: see if we could get case sensitive updates to downstream service / VAProfile
+    if (response?.[PERSONAL_INFO_FIELD_NAMES.PREFERRED_NAME]) {
+      set(
+        response,
+        PERSONAL_INFO_FIELD_NAMES.PREFERRED_NAME,
+        capitalize(response?.[PERSONAL_INFO_FIELD_NAMES.PREFERRED_NAME]),
+      );
+    }
+
+    dispatch({
+      type: FETCH_PERSONAL_INFORMATION_SUCCESS,
+      personalInformation: response,
+    });
+  };
+}
+
+// since the personal information api requests do no fall into a transactional life cylce
```

Comment:
DISCUSSION POINT:  For the transactional life cycle, is this API a one-off of the transactional paradigm, or are the API calls, in general, going in that direction? 

More concise question: Do we need to start looking at the UI to deprecate the transactional UI?

Reply:
So for right now we still have transactional requests with a 'polling' to a status endpoint for the address, phone, number, and email type fields. |

I don't think those are going away, and the personal information is essentially a one off set of profile fields that use a more straightforward sync pattern via vets api. Maybe we should eventually diverge into a different pattern in Redux for these types of api requests instead of using the transactional actions?